.circleci/config.yml

```diff
@@ -106,6 +106,17 @@ jobs:
       - run: sudo apt-get -y install curl musl-tools
       - run: curl https://sh.rustup.rs -sSf | sh -s -- -y
       - run: . "$HOME/.cargo/env"; rustup target add x86_64-unknown-linux-musl
+      # Install quality tools
```

Comment:
**issue:** Can we add the fact of putting the build in error if we have warnings during compilation?